pyomo/contrib/pynumero/plugins.py

```diff
@@ -25,15 +27,27 @@ def load():
         doc='Cyipopt: direct python bindings to the Ipopt NLP solver'
     )(PyomoCyIpoptSolver)
     SolverFactory.register(
-        "fsolve",
+        "scipy.fsolve",
```

Comment:
Does it make sense to leave a deprecation path (a registration to a simple deprecation wrapper around this solver) under the old name -- at least for a release or two?

Reply:
Yes, however, these solvers have not been part of a release yet. They were introduced in #2523 on Oct 5, and the last release I believe was in August. If this PR does not make the upcoming release, I will add a deprecation path.